tests/integration/static_ip_address_test.py

```diff
@@ -179,3 +181,70 @@ def test_add_iface_with_same_static_ipv4_address_to_existing(setup_eth1_ipv4):
     netapplier.apply(copy.deepcopy(desired_state))
 
     assertlib.assert_state(desired_state)
+
+
+def test_add_static_ipv6_with_full_state():
+    desired_state = statelib.show_only(('eth1',))
+    eth1_desired_state = desired_state[INTERFACES][0]
+
+    eth1_desired_state['state'] = 'up'
+    eth1_desired_state['ipv6']['enabled'] = True
+    eth1_desired_state['ipv6']['address'] = [
+        {'ip': '2001:db8:2::1', 'prefix-length': 64},
+        # This sequence is intentionally made for IP address sorting.
+        {'ip': '2001:db8:1::1', 'prefix-length': 64},
+    ]
+    netapplier.apply(copy.deepcopy(desired_state))
```

Comment:
not: For readability, please separate between setup, production call/execution and assertion (and teardown in needed) using newlines.
* For all tests added.

Reply:
You did it everywhere except this line :)